Makefile.Common

```diff
@@ -81,6 +81,11 @@ ALL_SRC := $(shell find $(ALL_PKG_DIRS) -name '*.go' \
                                 -not -path '*/local/*' \
                                 -type f | sort)
 
+ALL_SRC_AND_SHELL := $(shell find $(ALL_PKG_DIRS) -name '*.go' -o -name "*.sh" \
+                                -not -path '*/third_party/*' \
```

Comment:
```suggestion
                                -not -path '**/third_party/*' \
```
We may want to keep the same regex here for `third_party` directories, it looks like the `**` originally there allows crossing directory boundaries, so any `third_party` sub-directory would be ignored, not just the top level. [Relevant link](https://stackoverflow.com/questions/46547540/meaning-of-a-double-star-in-a-file-path). Let me know if I'm missing something here.

Reply:
Fair point, I missed that this was consistent with other references in the makefile. In that case, it's fine with me to leave it for now, but I'll defer to others if there are more concerns here. 👍